Conch/lib/Conch/Model/ValidationState.pm

```diff
@@ -20,6 +20,22 @@ use constant {
 my $attrs = [qw( id device_id validation_plan_id status created completed )];
 has $attrs;
 
+=head2 TO_JSON
+
+=cut
+
+sub TO_JSON ($self) {
+	{
+		id                 => $self->id,
+		device_id          => $self->device_id,
+		validation_plan_id => $self->validation_plan_id,
+		status             => $self->status,
+		created            => Conch::Time->new( $self->created )->rfc3339,
+		completed          => $self->completed
+			&& Conch::Time->new( $self->completed )->rfc3339
```

Comment:
As of 901f47a95bbfa7a36a5af65f39cd29784a758c9b, you don't need to do the rfc3339 conversion yourself. Conch::Time has a TO_JSON method and if everyone just uses that, we can guarantee a consistent representation.

Reply:
Cool! Will remove